packages/server/src/arpa_reporter/services/records.js

```diff
@@ -125,6 +125,10 @@ async function loadRecordsForUpload (upload) {
       const formattedRow = {}
       Object.keys(row).forEach(fieldId => {
         let value = row[fieldId]
+        if (rulesForCurrentType[fieldId] === null) {
+          // No known rules for this type, so we can't format it.
+          return;
+        }
```

Comment:
Cool, thanks for fixing this!

I guess now that we are reading field IDs from the sheet, they don't necessarily all exist in the committed ruleset.  Another oversight of mine.  😬